frontend/src/page/GameWizard.re

```diff
@@ -40,17 +40,17 @@ let make = () => {
   let (sprites, setSprites) = React.useState(() => []);
 
   let sendLogo = (): PromiseUtils.t(option(string)) =>
-    logo <$> (FileRestEndpoint.sendFile ||> Repromise.mapOk(x => Some(x))) ||? PromiseUtils.resolved(None);
+    logo <$> (FileRestEndpoint.sendFile ||> (p => Promise.mapOk(p, x => Some(x)))) ||? PromiseUtils.resolved(None);
```

Comment:
I think what I've done here, at `(p => ...)` is questionable. I didn't want to remove the use of `||>`. I'll be happy to change this.